### DISCUSSION TITLE
use as standard javascript module

### DISCUSSION THREAD

**Discussion by dbuerer** [josdejong/svelte-jsoneditor#155] [Q&A] [answered]
i think i'm kind-of a noob to what's going on here...but i really just am trying to load this as a standard javascript module.  

Where are the JS files that i need to load?

**Comment by josdejong** [comment, marked as answer]
You have to install ("download") the files via [npm](npmjs.com/), or use a CDN like:

- jsdeliver: https://www.jsdelivr.com/package/npm/vanilla-jsoneditor
- unpkg: https://unpkg.com/vanilla-jsoneditor/

Documentation:

How to install: https://github.com/josdejong/svelte-jsoneditor#install
Use in standard JavaScript: https://github.com/josdejong/svelte-jsoneditor#standalone-bundle-use-in-react-vue-angular-plain-javascript-

**Reply by dbuerer** [reply]
thank you.  that got me going and i'm up and running.  love this library it's really cool!
